lib/services/presets.go

```diff
@@ -733,13 +737,40 @@ func NewPresetTerraformProviderRole() types.Role {
 					types.NewRule(types.KindGitServer, RW()),
 					types.NewRule(types.KindAutoUpdateConfig, RW()),
 					types.NewRule(types.KindAutoUpdateVersion, RW()),
+					types.NewRule(types.KindHealthCheckConfig, RW()),
```

Comment:
I think we need to update the role editor to incorporate this as well, don't we?